semgrep/semgrep/cli.py

```diff
@@ -108,6 +108,11 @@ def cli() -> None:
         action="store_true",
         help="Scan all files even those ignored by a projects gitignore(s)",
     )
+    parser.add_argument(
+        "--ignore_unknown_extentions",
```

Comment:
```suggestion
        "--ignore-unknown-extensions",
```

(will require changes later as well)